Gulpfile.js

```diff
@@ -64,6 +67,12 @@ gulp.task('deploy', ['build'], () =>
 
 gulp.task('deploy-test', ['build'], () =>
     gulp.src(files.dist + '/**/*')
+        .pipe(gulpIf(args.appId, replace(
+            /window\.BinaryBoot\.appId = window\.cordova \? 1006 : 1001;/,
+            'window.BinaryBoot.appId = ' + args.appId + ';',
+            { skipBinary: true }
+            )
```

Comment:
This is added so that OAuth redirect correctly, 
eg.

`gulp deploy-test --appId 1002`

Acceptable? If so I'll update documentations